src/python/grpcio_tests/tests/unit/_reconnect_test.py

```diff
@@ -88,6 +89,7 @@ def test_reconnect(self):
         multi_callable = channel.unary_unary(_UNARY_UNARY)
         self.assertEqual(_RESPONSE, multi_callable(_REQUEST))
         server.stop(None)
+        time.sleep(1)
```

Comment:
Please add a comment explaining this. And a TODO to remove this once retries are added. Otherwise this hack will stay forever and we don't want that.

Reply:
I have filed an issue for it.